pkg/istio-agent/agent.go

```diff
@@ -91,11 +91,11 @@ type Agent struct {
 	localDNSServer *dns.LocalDNSServer
 }
 
-// AgentConfig contains additional config for the agent, not included in ProxyConfig.
+// AgentOptions contains additional config for the agent, not included in ProxyConfig.
 // Most are from env variables ( still experimental ) or for testing only.
 // Eventually most non-test settings should graduate to ProxyConfig
 // Please don't add 100 parameters to the NewAgent function (or any other)!
-type AgentConfig struct {
+type AgentOptions struct {
```

Comment:
Should we just move them to ProxyConfig and get rid of the extra options ?

Reply:
I think this was to make it just own only the necessary fields